importer.js

```diff
@@ -5,12 +5,22 @@ const parse = require('csv-parse');
 const transform = require('stream-transform');
 
 
-function parseValue(recordValue, mappingObject) {
+function parseValue(recordObject, recordKey, mappingObject) {
```

Comment:
I believe that we should use `timestamp` how we used to. No necessary to add `recordObject` here.